packages/neon-core/__tests__/wallet/Balance.ts

```diff
@@ -23,9 +26,10 @@ describe("constructor", () => {
       assetSymbols: ["ASS"],
       tokenSymbols: ["TOK"],
       assets: {
-        ASS: {} as AssetBalanceLike,
+        ASS: { hash: NEO_ASSET_HASH } as AssetBalanceLike,

```

Comment:
Can we not use the name NEO_ASSET_HASH if it is not? This is confusing.

Reply:
I agree, I changed it to DEFAULT_HASH